integration_tests/steps/load_tests.py

```diff
@@ -95,19 +95,16 @@ def check_min_user_rate_profile_dict(context):
     load1 = next(filter(lambda x: x.name == "H1 General Load 1", house.children))
     load2 = next(filter(lambda x: x.name == "H1 General Load 2", house.children))
 
-    num_of_hierarchies = 2
     for market in house.past_markets:
         assert len(market.trades) > 0
         for trade in market.trades:
             trade_rate = trade.offer.price / trade.offer.energy
             if trade.buyer == load1.name:
                 min_rate = load1.strategy.initial_buying_rate[market.time_slot]
-                assert trade_rate * (1 + context.simulation.simulation_config.iaa_fee /
```

Comment:
Can you explain why was that removed ?

Reply:
This is probably due to the fact that the iaa_fee is 0 if not set, right @faizan2590 ? For this Integration test setup it is not set, IIRC.

Reply:
EDIT: Sorry, no, I misread. The iaa_fee is set to the global on if not set explicitly. So, yes, please explain. Thanks!